package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-jw-media-player",
-  "version": "0.2.26",
+  "version": "0.2.3",
```

Comment:
This should be `"version": "0.2.27"`

Reply:
@archcorsair Thanks for pointing this out, all updated.